From 7b9212cef4858f2a3f9158679c128be4bed65732 Mon Sep 17 00:00:00 2001
From: William Jon McCann <jmccann@redhat.com>
Date: Mon, 02 Jul 2012 16:04:04 +0000
Subject: Don't show open with for folders

The only thing that handles this by default is nautilus. The
same effect is better done through select all in a folder.
---
diff --git a/src/nautilus-view.c b/src/nautilus-view.c
index 7acda78..3044c2c 100644
--- a/src/nautilus-view.c
+++ b/src/nautilus-view.c
@@ -4575,9 +4575,8 @@ reset_open_with_menu (NautilusView *view, GList *selection)
 
 		file = NAUTILUS_FILE (node->data);
 
-		other_applications_visible &= ((!nautilus_mime_file_opens_in_view (file) &&
-						!nautilus_file_is_nautilus_link (file)) ||
-					       nautilus_file_is_directory (file));
+		other_applications_visible &= (!nautilus_mime_file_opens_in_view (file) &&
+					       !nautilus_file_is_nautilus_link (file));
 	}
 
 	default_app = NULL;
--
cgit v0.9.0.2
